### DISCUSSION TITLE
Contacts, groups and company names

### DISCUSSION THREAD

**Discussion by bluikko** [netbox-community/netbox#11018] [Q&A]
With the upcoming removal of the deprecated `[...]_contact` fields in favor of `Contact` objects I have been looking at how to migrate that information. The first obvious thing that popped up is how to record "company" information.  
How are people recording that information? There is no suitable field in the `Contact` object. Is `Contact Group` supposed to be used for this? Some naming convention for the `Name` attribute of `Contact` object? A custom field of `Contact` object?

Edit: finally understood how to assign Contacts and Contact Roles.. phew.

**Comment by mtinberg** [comment]
If you have multiple contacts at a particular company, then a Contact Group for the company and Contacts for the individuals (and role accounts like ticket emails) seems to make sense to me.

https://demo.netbox.dev/tenancy/contact-groups/2/


—

Mark Tinberg ***@***.***>

Division of Information Technology-Network Services

University of Wisconsin-Madison


________________________________
From: bluikko ***@***.***>
Sent: Thursday, November 24, 2022 4:23 AM
To: netbox-community/netbox ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [netbox-community/netbox] Contacts, groups and company names (Discussion #11018)


With the upcoming removal of the deprecated [...]_contact fields in favor of Contact objects I have been looking at how to migrate that information. The first obvious thing that popped up is how to record "company" information.
How are people recording that information? There is no suitable field in the Contact object. Is Contact Group supposed to be used for this? Some naming convention for the Name attribute of Contact object? A custom field of Contact object?
